### PR TITLE
Ca feature/ec2 deploy and some changes

### PR DESCRIPTION
**Pull request by cagodoy**
- [X] Implement 'callback_post_url' using pigeon-http service inside scheduler.go file (ca)
- [X] Deploy in EC2 Server (ca)
- [x] Configure Domain api.iampigeon.com (ca)

### CODE REVIEW

File: pigeon.go [hunk 1]
```diff
@@ -131,7 +143,7 @@ type MQTT struct {
 type HTTP struct {
 	URL     *url.URL               `json:"url"`
 	Body    string                 `json:"body,omitempty"`
-	Headers map[string]interface{} `json:"headers"`
+	Headers map[string]interface{} `json:"headers,omitempty"`
```
Comment (mtavano) [inline pigeon.go:146]: Headers must be `map[string]string`

File: pigeon.go [hunk 2]
```diff
@@ -189,3 +201,10 @@ type Mock struct {
 	// SubjectsChannels []*SubjectsChannels `json:"subjects_channels"`
 	//Messages         []*Message          `json:"messages"`
 }
+
+// Response ...
+type Response struct {
+	Data  interface{} `json:"data,omitempty"`
+	Meta  interface{} `json:"meta,omitempty"`
+	Error interface{} `json:"meta,omitempty"`
```
Comment (mtavano) [inline pigeon.go]: Must change the json tag

```go
type Response struct {
            Data        interface{} `json:"data, omitempty`
            Meta       interface{} `json:"meta, omitempty`
            Error       interface{} `json:"error, omitempty`
}
```

File: scheduler/scheduler.go [hunk 3]
```diff
@@ -81,6 +83,14 @@ func (s *service) Put(id ulid.ULID, content []byte, endpoint pigeon.NetAddr, sta
 			return e
 		}
 
+		// send http error via pigeon-htpp
```
Comment (mtavano) [inline scheduler/scheduler.go]: _Send http error **through** pigeon-http_

**Review by mtavano — CHANGES_REQUESTED** [review]

File: pigeon.go (hunk 1 above)
Comment (cagodoy) [reply pigeon.go:146]: ok

**Pull request merged**